### PR TITLE
(WIP) Adds a new surgical module for medborgs- the Advanced Robotic Manipulator

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Medborgs now have a "hand" for use in surgeries that require the use of hands.

### To Do:

- [ ] Get cavity implant surgery working with this.

## Why It's Good For The Game
This should fix problems with borgs being unable to do surgical steps requiring hands.

## Changelog
:cl:
add: Medical borgs now have a new module: the Advanced Robotic Manipulator, or ARM. This is used for surgical steps that require hands.
/:cl:

### CODE REVIEW

File: code/modules/surgery/stomachpump.dm [hunk 1]
```diff
@@ -25,29 +25,29 @@
 //Working the stomach by hand in such a way that you induce vomiting.
 /datum/surgery_step/stomach_pump
 	name = "Pump Stomach"
-	accept_hand = TRUE
+	implements = list(/obj/item/reagent_containers/syringe = 100)
```
Comment (ATH1909) [inline code/modules/surgery/stomachpump.dm]: can you add ghetto surgery equivalents for this step?

**Comment by itseasytosee** [comment]
So now instead of inducing post-mortem vomit your just taking it out with a tiny syringe very slowly?

File: code/modules/surgery/stomachpump.dm (hunk 1 above)
Comment (Iatots) [reply code/modules/surgery/stomachpump.dm]: Stab with pen and succ, 5% of reagents transfered in your stomach.

**Comment by ArcaneMusic** [comment]
Fixes part of #48941.

File: code/modules/surgery/stomachpump.dm (hunk 1 above)
Comment (EOBGames) [reply code/modules/surgery/stomachpump.dm]: I'd been struggling trying to think of a good ghetto equivalent- if you have a suggestion, I'd be happy to add it.

Comment (itseasytosee) [reply code/modules/surgery/stomachpump.dm]: Rapid piping device clearly.

Comment (Dennok) [reply code/modules/surgery/stomachpump.dm]: Plunger?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/surgery/stomachpump.dm]: you should be able to use your hand but the reagents get on your hand (you get touch reacts from the chems you're pulling out) ;)

**Comment by ExcessiveUseOfCobblestone** [comment]
just give borgs an advanced manipulation tool that acts as a hand purely for the purpose of acting as a hand in surgery.

**Comment by MarioWizard119** [comment]
If the reagents extracted are put in the syringe, you’ll need to make a blacklist on extractable chems. Things like changeling adrenaline or healing peptides would need to delete themselves if extracted, or else it could be easily abused with an Odysseus, which is why many extracting surgeries were never merged.

**Comment by EOBGames** [comment]
Boy the comments on this are gonna be confusing now.

**Comment by tiramisuapimancer** [comment]
Does the hugging module not work as hands for certain surgeries? It absolutely works for things like Vein Muscle Membrane, and shaking people to get them up.

**Comment by ATH1909** [comment]
> Does the hugging module not work as hands for certain surgeries? It absolutely works for things like Vein Muscle Membrane, and shaking people to get them up.

It, uh, shouldn't. It gives people a MESSAGE that you're shaking them up, but doesn't actually help them get up faster, unlike when a human shakes someone up.

**Comment by tiramisuapimancer** [comment]
> It, uh, shouldn't. It gives people a MESSAGE that you're shaking them up, but doesn't actually help them get up faster, unlike when a human shakes someone up.

Not for getting people up faster when they can't get themselves up, it does get them up if they were simply resting. And it does still work as a hand for surgery steps that you use your hands with.

**Comment by ExcessiveUseOfCobblestone** [comment]
if the hug module already does this just call us dumb and we can go home

**Comment by trollbreeder** [comment]
@ExcessiveUseOfCobblestone 
https://www.youtube.com/watch?v=KnurvMjfpUA
Turns out, yes, the hugging module works as a hand in surgeries. This is demonstrated with Nerve Grounding in the video.

Anyhoo, it's normal for a borg to be unable to do something by not being able to pick something up. Think engiborg and plasma tanks. We can go home now.

**Pull request closed without merging**